Source/Core/Core/CoreParameter.h

```diff
@@ -201,6 +204,15 @@ struct SCoreStartupParameter
 	int iHotkey[NUM_HOTKEYS];
 	int iHotkeyModifier[NUM_HOTKEYS];
 
+	//Remove Layer
+	std::vector<SkipEntry> object_removal_codes;
+	u32 skip_objects_end = 0;
+	u32 skip_objects_start = 0;
+	size_t num_object_removal_codes = 0;
+	std::vector<int> num_object_removal_data_bytes;
+	volatile bool update = true;
+	volatile bool done = true;
```

Comment:
@degasus Makes sense, thanks!  Yesterday I had a couple minutes to start trying to change this over to mutexs, but was getting crashes, so I've obviously mucked something up.  I'm sure I'll figure it out, just need a bit more time.